modules/admin_manual/pages/configuration/user/oidc/oidc.adoc

```diff
@@ -43,7 +43,15 @@ Setting up ownCloud Server to work with OpenID Connect requires a couple of comp
 - An external identity provider configured to work with the ownCloud components
 - A distributed memcache setup - such as Redis or Memcached - is required to operate this app. Follow the xref:configuration/server/caching_configuration.adoc[caching documentation] on how to set it up.
 - The {oc-marketplace-url}/apps/openidconnect[OpenID Connect App] installed on ownCloud Server
-- Configuration for the OpenID Connect App in `config.php` on ownCloud Server
+- Configuration settings in `config.php` on ownCloud Server
++
+* `'http.cookie.samesite' \=> 'None',`
++
+see `config.sample.php` for more details.
++
+* Settings for the OpenID Connect App
```

Comment:
```suggestion
- Settings for the OpenID Connect App
```

This item is intriduced by "*" but the ones above start with "-". Does that make a difference?

Reply:
It is correct that it is different, because it tells antora that this is a different level.
Else it will not render as you see in the screenshot.

Reply:
I mark this as resolved